src/shared/logger/logger.ts

```diff
@@ -67,9 +67,7 @@ export function compareLogLevel(l1: LogLevel, l2: LogLevel): number {
 export function getLogger(type?: 'channel' | 'debugConsole' | 'main'): Logger {
     const logger = toolkitLoggers[type ?? 'main']
     if (!logger) {
-        throw new Error(
-            'Logger not initialized. Extension code should call initialize() from shared/logger/activation, test code should call setLogger().'
-        )
+        return new ConsoleLogger()
```

Comment:
No reason to throw if logger is not ready, it just means we should fallback to `console.log()`.